config/crd/bases/postgresql.cnpg.io_clusters.yaml

```diff
@@ -3909,6 +3909,18 @@ spec:
                           type: string
                       type: object
                     type: array
+                  tls:
+                    description: |-
+                      Configure TLS communication for the metrics endpoint.
+                      Changing tls.enable option will force a rollout of all instances.
+                    properties:
+                      enable:
```

Comment:
Why 'enable` when we already have `enabled` in other places?

Reply:
Because you are right, and this is a typo, nobody noticed.

Reply:
fix committed